stock_packaging_usability_ul/__manifest__.py

```diff
@@ -1,37 +1,23 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Stock Packaging Usability UL module for Odoo
-#    Copyright (C) 2014 Akretion (http://www.akretion.com).
-#    @author Alexis de Lattre <alexis.delattre@akretion.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright 2014-2018 Akretion (http://www.akretion.com).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Stock Packaging Usability UL',
-    'version': '8.0.1.0.0',
-    'category': 'Warehouse Management',
+    'version': '10.0.1.0.0',
+    'category': 'Warehouse',
     'license': 'AGPL-3',
-    'summary': "Faster packaging process with logistical units",
+    'summary': "Faster packaging process with logistics units",
     'author': 'Akretion,Odoo Community Association (OCA)',
     'website': 'http://www.akretion.com',
-    'depends': ['stock_packaging_usability'],
+    'depends': [
+        'stock_packaging_usability',
+        'stock_product_ul',
+        ],
     'data': [
```

Comment:
demo data are welcome

Reply:
Demo data are in stock_product_ul. Demo data doesn't make sense in the module stock_packaging_usability_ul